src/Swashbuckle.AspNetCore.SwaggerUI/index.js

```diff
@@ -57,6 +57,7 @@ window.onload = function () {
     if (interceptors.ResponseInterceptorFunction)
         configObject.responseInterceptor = parseFunction(interceptors.ResponseInterceptorFunction);
 
+    configObject.plugins = configObject.plugins ? configObject.plugins.map(eval) : [];
```

Comment:
Would it be better to omit setting it at all?

```suggestion
    if (configObject.plugins) {
        configObject.plugins = configObject.plugins.map(eval);
    }
```